### PR TITLE
Ortools 9.3 w/ GLPK

### PR DESCRIPTION
**Pull request by jashan-lco**
GLPK wheels are built and hosted at: https://github.com/observatorycontrolsystem/ortools-glpk-wheels

Helm changes to go with this: 
https://github.com/LCOGT/helm-charts/pull/101
https://github.com/LCOGT/eks-production1/pull/63

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,20 +72,32 @@ The scheduler can either be run directly on a machine, or in the provided Docker
...
 
-You can then update the container name in the supplied **docker-compose.yml** and run the scheduler using
+You can also use `docker-compose`:
 
-`cd deploy; docker-compose up`
+```shell
+$ ./docker-compose.sh up --build
+```
 
 To run the unit tests instead of the actual scheduler, change the commented out command in the **deploy/docker-compose.yml** file to run nosetests with the desired arguments.
```
Comment (jnation3406) [inline README.md:102]: nosetests -> pytest

File: adaptive_scheduler/cli.py [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
-from adaptive_scheduler.interfaces import NetworkInterface
-from adaptive_scheduler.observations import ObservationScheduleInterface
-from adaptive_scheduler.observation_portal_connections import ObservationPortalInterface
-from adaptive_scheduler.configdb_connections import ConfigDBInterface
-from adaptive_scheduler.scheduler import LCOGTNetworkScheduler, SchedulerRunner
-from adaptive_scheduler.scheduler_input import SchedulingInputFactory, SchedulingInputProvider, \
-    FileBasedSchedulingInputProvider, SchedulerParameters
-from adaptive_scheduler.monitoring.network_status import Network
-from adaptive_scheduler.kernel.fullscheduler_ortoolkit import FullScheduler_ortoolkit, ALGORITHMS
+from .eventbus import get_eventbus
```
Comment (jnation3406) [inline adaptive_scheduler/cli.py:9]: did you need to switch to relative imports? I think I prefer absolute imports, but i guess its just preference nowadays...

File: adaptive_scheduler/cli.py [hunk 3]
```diff
@@ -77,13 +80,15 @@ def parse_args(argv):
     arg_parser.add_argument("-f", "--fromfile", type=str, dest='input_file_name', default=defaults.input_file_name,
                             help="Filename for scheduler input. Example: -f scheduling_input_20180101.pickle")
     arg_parser.add_argument("-g", "--mip_gap", type=float, default=defaults.mip_gap,
-                            help="The acceptable MIP GAP threshold used in the solver. Defaults to 0.01 (1%). Recommended range 0.01-0.0001")
+                            help="The acceptable MIP GAP threshold used in the solver. Defaults to 0.01 (1%%). Recommended range 0.01-0.0001")
```
Comment (jnation3406) [inline adaptive_scheduler/cli.py:83]: Is this escaping the %?

**Review by jnation3406 — APPROVED** [review]
Looks good to me. Great work updating to pytest and latest python too!

File: adaptive_scheduler/cli.py (hunk 2 above)
Comment (jashan-lco) [reply adaptive_scheduler/cli.py:9]: No didn't have to, just a force of habit. I can change to absolute for consistency, but at some point let's start running our projects through a code formatting tool like https://github.com/google/yapf.

**Pull request merged**